### PR TITLE
add support for 1.20.5-RC1

### PR DESCRIPTION
**Pull request by Outfluencer**

### CODE REVIEW

File: protocol/src/main/java/net/md_5/bungee/protocol/packet/LoginSuccess.java [hunk 1]
```diff
@@ -37,6 +37,11 @@ public void read(ByteBuf buf, ProtocolConstants.Direction direction, int protoco
         {
             properties = readProperties( buf );
         }
+        // remove in next minecraft release
```
Comment (md-5) [inline protocol/src/main/java/net/md_5/bungee/protocol/packet/LoginSuccess.java:40]: you should at least comment what it is

File: protocol/src/main/java/net/md_5/bungee/protocol/packet/LoginSuccess.java [hunk 2]
```diff
@@ -54,6 +59,12 @@ public void write(ByteBuf buf, ProtocolConstants.Direction direction, int protoc
         {
             writeProperties( properties, buf );
         }
+        // remove in next minecraft release
```
Comment (md-5) [inline protocol/src/main/java/net/md_5/bungee/protocol/packet/LoginSuccess.java:62]: you should at least comment what it is

Comment (Outfluencer) [reply protocol/src/main/java/net/md_5/bungee/protocol/packet/LoginSuccess.java:64]: done

Comment (md-5) [reply protocol/src/main/java/net/md_5/bungee/protocol/packet/LoginSuccess.java:64]: Thanks

**Pull request closed without merging**